stdlib/public/core/StringGuts.swift

```diff
@@ -309,16 +313,17 @@ extension _StringGuts {
   @_inlineable
   internal var _unmanagedRawStart: UnsafeRawPointer {
     @inline(__always) get {
-      _sanityCheck(_object.isUnmanaged)
-      return _object.asUnmanagedRawStart
+      _sanityCheck(_object.isUnmanagedOrNative)
+      return _object.asUnmanagedRawStart + 32
```

Comment:
Should this be `+ _pointerBias`?

Reply:
Doh, you're right. I also should brush up the comments.

Reply:
Fixed in amended commit